reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,88 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.22.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.22.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+* Support for 64-bit only systems for all supported Android versions.<!--AC-1890, AC-2428-->
+* Support for installing relevant 32-bit NVIDIA userspace binaries on AMD64 systems.<!--AC-2450-->
+* To allow easier identification and use of instances, you can now explicitly name an instance when creating it. See [How to create an instance](https://anbox-cloud.io/docs/howto/instance/create).<!--AC-2395-->
+* Support for Vulkan 1.3 for Android 13 images.<!--AC-2278-->
```

Comment:
@keirthana just another headup, the support for Vulkan 1.3 to Android 13 images is not there yet for 1.22.0 due to a bug we found. The [fix](https://code.launchpad.net/~morphis/indore/+git/mesa/+merge/465849) won't be included in 1.22.0 release but 1.22.1 release as discussed with @morphis

Reply:
Thanks Gary, I will remove this for 1.22.0.